great_tables/_utils.py

```diff
@@ -136,7 +136,7 @@ def _object_as_dict(v: Any) -> Any:
         return list(_object_as_dict(i) for i in v)
     if isinstance(v, dict):
         return dict((k, _object_as_dict(val)) for (k, val) in v.items())
-    if type(v) == type(_object_as_dict):  # FIXME figure out how to get "function"
+    if type(v) is type(_object_as_dict):  # FIXME figure out how to get "function"
```

Comment:
This is to satisfy newer flake8: https://github.com/amol-/great-tables/actions/runs/11259168945/job/31307498137

```
great_tables/_utils.py:139:8: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
```
but indeed it was a good idea anyway

Reply:
It seems like maybe this check was trying to do either `if callable(v):` or maybe `inspect.isfunction(v)` 😓